cloud/passportauth/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@raincatcher/passport-auth",
```

Comment:
WDYT about`auth-passport`? This will help us to do some operations using lerna filter `auth-*`